events/code_review_events/bugbug_utils.py

```diff
@@ -327,24 +335,40 @@ def add_new_jobs(self, revision, selected_tasks):
             # revision from the decision task).
             taskGroupId = status["taskGroupId"]
             try:
-                build = self.task_group_to_build.get(taskGroupId)
+                push = self.task_group_to_push.get(taskGroupId)
+                build = push["build"]
+                revision = push["revision"]
```

Comment:
We could possibly get the revision from other_routes (e.g. route.tc-treeherder.v2.try.028980a035fb3e214f7645675a01a52234aad0fe.455891) to avoid storing the revision too on Redis, but this might be more robust (what if the tc-treeherder route is removed?) for a very small price.

Reply:
Fine with me.